wiki/Modding/Nuke/en.md

```diff
@@ -0,0 +1,19 @@
+# Nuke
+
+**Nukes** were a [beatmap](/wiki/Beatmaps) moderation tool in the early years of osu!. Any beatmaps that were considered to be disregarding the rules (or the mapper disregarding the [Code of Conduct](/wiki/Ranking_Criteria/Code_of_Conduct)) could be nuked by either the [Beatmap Appreciation Team](/wiki/Modding/Beatmap_Appreciation_Team/) or the [Global Moderation Team](/Wiki/People/The_Team/Global_Moderation_Team/).
+
+When a beatmap was nuked it would have a radioactive symbol ![](/wiki/shared/icon/nuke.gif) on its beatmap thread, signaling to other members of the BAT/GMT that it was not to be considered for ranked until significant changes had been made. If a beatmap made the corresponding changes, the icon could be removed and it could proceed with the [ranking process](/wiki/Beatmap_ranking_procedure/).
```

Comment:
```suggestion
When a beatmap was nuked, it would have a radioactive symbol ![](/wiki/shared/icon/nuke.gif) on its beatmap thread. This would signal other members of the BAT/GMT that it was not to be considered for [Ranked](/wiki/Beatmap_ranking_procedure#ranked) until significant changes had been made. If a beatmap made the corresponding changes, the icon would be removed and it could proceed with the [ranking process](/wiki/Beatmap_ranking_procedure).
```